Week1/HTML.md

```diff
@@ -0,0 +1,260 @@
+
+    <details>
+     <summary>Code result</summary>
+       &lt;div&gt;
+       <br>
+        &lt;h1&gt;The Main Heading&lt;/h1&gt;
+        <br>
+        &lt;!-- My smart comment! --&gt;
+        <br>
+        &lt;h2&gt;Subheading&lt;/h2&gt;
+        <br>
+        &lt;p&gt;Paragraph&lt;/p&gt;
+        <br>
+        &lt;img src="/" alt="Image"&gt;
+        <br>
+        &lt;div&gt;
+        <br>
+          &lt;p&gt;Second Paragraph&lt;/p&gt;
+          <br>
+        &lt;/div&gt;
+        <br>
+      &lt;/div&gt;
+    </details>
+
+      <details>
+      <summary>Result in browser</summary>
+        <div>
+          <h1>The Main Heading</h1>
+          <h2>Subheading</h2>
+          <p>Paragraph</p>
+          <img src="/" alt="Image">
+          <div>
+            <p>Second Paragraph</p>
+          </div>
+        </div>
+      </details>
+      
+### Task 2: Add more elements to your HTML page  
+
+- Read about ways of creating [ordered](https://www.w3schools.com/tags/tag_ol.asp) and [unordered](https://www.w3schools.com/tags/tag_ul.asp) lists in html. In your file add two new `div`s and create one list in each of them. The lists should look like this:  
+
+My to-do list:
+1. (_insert your tasks here_) 
+2.  
+3.  
+
+  Three people sitting next to me:  
+  - (_insert their names_)
+  -  
+  -  
+
+### Task 3 - Build a webpage to match the mockup  
+
+Create a new HTML file (let's call it exercise.html) and try to create a webpage that will look exactly like this mockup (you can click on the image to enlarge it). Mind the instructions.
+![](https://i.imgur.com/UsMwmXa.png)
+
+##### Instructions  
+- Use only HTML, no CSS.
+- You can choose another picture of Narhwal, this picture is just an example.
+- The picture should be a link leading to an article about Narwhal (use whatever article you would like).
+- The name _Mac's Mission_ should lead to https://www.iampitbullstrong.com//.  
+
+<details>
+     <summary>Hints</summary>  
+  
+  - Read about HTML [links](https://www.w3schools.com/html/html_links.asp)  
+  
+  - Read about HTML [tags](https://www.w3schools.com/tags/tag_hr.asp) (`<em>`, `<strong>`, `<hr>`, `<sub>`, `<s>`)  
+  
+  - Read about HTML [forms](https://www.w3schools.com/html/html_forms.asp)  
+  
+  </details>
+  
+<details>
+     <summary>See the solution (only when done!)</summary>  
+  
+  https://codepen.io/kira-abl/pen/mddYrjj?editors=1000
+  
+</details>
+
+### Task 4 - More HTML tags
+
+**Tables**
+
+In html a table is defined with the `<table>` tag. Each row is defined with the `<tr>` tag (**t**able **r**ow). A **t**able **h**eader is defined with the `<th>` tag. Each cell is defined with the `<td>` (**t**able **d**ata) tag.
+
+* Read more about [tables](https://www.w3schools.com/html/html_tables.asp), in a new .html file build a table that will have the following information in it:
+
+![](https://i.imgur.com/u9jlXqU.png)
+
+**HTML `<meta>` Tag**
```

Comment:
this is an advanced topic, add it in the `Advanced Topics` section as a link.

Reply:
Do you mean a link in the section or a link in general?
I think the description should stay but under a heading of advanced

Reply:
Yeah an Advanced heading would be great instead of being in `Task 4`. 
But then you dont need the description just have a title and the link. 
But if you want to keep the description then fine.

Reply:
Add the heading and keep the description :+1:

Reply:
Okay, those are now under Advanced. The descriptions and instructions are there.